### PR TITLE
Issue #11214: Specified violation messages in ImportOrderCheck

### PR DESCRIPTION
**Pull request by jxr98**
#11214 specified violation messages in ImportOrderCheck.

### CODE REVIEW

**Comment by jxr98** [comment]
Hi, it looks like permission is required for one github action to run. Also I don't know why this ci/semaphoreci/pr action failed. Any help would be appreciated.

**Comment by romani** [comment]
Travis is system problem, semaphore will be fixed tomorrow by another PR

File: src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/importorder/InputImportOrder2.java [hunk 1]
```diff
@@ -18,20 +18,20 @@
...
 import static java.io.File.createTempFile; // ok
-import java.io.IOException; // violation
+import java.io.IOException; // violation 'Wrong order for 'java.io.IOException' import.'
 import java.io.InputStream; // ok
 import java.io.Reader; // ok
-import static javax.swing.WindowConstants.*; // violation
+import static javax.swing.WindowConstants.*; // violation 'Wrong order for 'javax.swing.WindowConstants.*' import.'
 
-import static sun.tools.util.ModifierFilter.ALL_ACCESS; // violation
+import static sun.tools.util.ModifierFilter.ALL_ACCESS; // violation 'Extra separation in import group before 'sun.tools.util.ModifierFilter.ALL_ACCESS''
```
Comment (romani) [inline src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/importorder/InputImportOrder2.java:34]: I am not sure why Line length validation is not complaining.
But let's keep messaging on limit 120 symbols
`Extra separation in import group before .*`

Please apply same for others long lines.

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/importorder/InputImportOrder2.java (hunk 1 above)
Comment (jxr98) [reply src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/importorder/InputImportOrder2.java:34]: Thanks for reviewing

**Review by romani — APPROVED** [review]
ok to merge

**Comment by romani** [comment]
@jxr98 , thanks a lot for PR, please do more.

**Comment by jxr98** [comment]
No problem. Will explore more issues.

**Pull request merged**